src/NServiceBus.Msmq.AcceptanceTests/When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx.cs

```diff
@@ -12,12 +11,17 @@ public class When_Audit_OverrideTimeToBeReceived_set_and_native_receive_tx : NSe
         [Test]
         public async Task Endpoint_should_not_start_and_show_error()
         {
-            var context = await Scenario.Define<Context>()
-                .WithEndpoint<Endpoint>()
-                .Done(c => c.EndpointsStarted)
-                .Run();
-
-            Assert.True(context.Exceptions.First().Message.Contains("Setting a custom OverrideTimeToBeReceived for audits is not supported on transactional MSMQ."));
+            try
```

Comment:
This is because `Assert.Throws` don't play ball with async in the nunit version we use right?